### PR TITLE
Cloud formation

### PR DESCRIPTION
**Pull request by pallenSL**
Docker Swarm deployment pipeline.

### CODE REVIEW

File: Comms/app.config [hunk 1]
```diff
@@ -1,9 +1,9 @@
-﻿<configuration>
-  <appSettings>
-    <add key="BaseUrl" value="http://*:8080"/>
-    <add key="AWSProfileName" value="dynamo-db-write"/>
```
Comment (jebbinSCL) [inline Comms/app.config]: Why is the awsProfileName no longer required?

Comment (pallenSL) [inline Comms/app.config]: Secrets are all out of GIT now.  Mounted into the container.

Comment (jebbinSCL) [inline Comms/app.config]: What about for local development? What's the setup change for that to continue working?

Comment (pallenSL) [inline Comms/app.config]: A local secrets.config file.

Comment (jebbinSCL) [inline Comms/app.config]: Ah, I missed the change to the appSettings node.

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**